stdlib/public/core/DoubleWidth.swift.gyb

```diff
@@ -40,9 +41,9 @@
 ///     }
 ///     // Prints "Too big to be an 'Int'!"
 ///
-/// The `DoubleWidth` type is intended for intermediate calculations, not as a
-/// replacement for a variable-width integer type. Nesting `DoubleWidth`
-/// instances, in particular, can result in undesirable performance.
+/// The `DoubleWidth` type is not intended as a replacement for a variable-width
+/// integer type. Nesting `DoubleWidth` instances, in particular, may result in
+/// undesirable performance.
```

Comment:
To me "undesirable performance" sounds like "you'll get too much performance even if you don't need it" ;-)

Reply:
After I’m done with it, maybe you will... ;)